app/helpers/application_helper.rb

```diff
@@ -90,6 +91,8 @@ def country_cover(country, with_tag: true)
   end
 
   def region_cover(region, with_tag: true)
+    version = Rails.application.secrets.mapbox[:version]
```

Comment:
I was looking for an equivalent to `before_action` for helpers that would allow you to set the version for each of the methods that require it, or at least a private method, but this isn't a controller. It's not actually that necessary and it likely doesn't exist but just for my own understanding. 

Also, saw a TODO about passing CSS classes to params? So you want to be able to extract the CSS class from the params, like `params[:css_class]` for example?

Reply:
Yeah given this is not a controller it would require the class to be rewritten a bit perhaps.

With regards the comment, I think that's a very old one (before my time) so I think I'll remove it.

Thanks!